src/map/GMap.js

```diff
@@ -455,6 +649,66 @@
     GMap.prototype.zoomToFit = function () {
         return this.zoomTo(this.data());
     };
+    
+    GMap.prototype.drawingOptions = function(_) {
+        if(typeof(_) === "object") {
+            if (typeof(this._drawingManager) === "object") {
+                this._drawingManager.setOptions(_);
+            }
+        }
+        return this._drawingManager;
+    };
```

Comment:
There is no getOptions method in on the drawing manager object in the google maps API.  Returning the drawing manager object itself though provides an object that encapsulates all of the drawing options currently specified.

I can re-factor this to do that if it's acceptable.